tests/contrib/open_telemtry/test_open_telemetry_middleware.py

```diff
@@ -0,0 +1,129 @@
+from time import sleep
+from typing import Any, Tuple, cast
+
+from opentelemetry.metrics import get_meter_provider, set_meter_provider
+from opentelemetry.sdk.metrics._internal import MeterProvider
+from opentelemetry.sdk.metrics._internal.aggregation import (
+    ExplicitBucketHistogramAggregation,
+)
+from opentelemetry.sdk.metrics._internal.export import InMemoryMetricReader
+from opentelemetry.sdk.metrics._internal.instrument import Counter
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import Span, TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
+
+from starlite import WebSocket, create_test_client, get, websocket
+from starlite.contrib.open_telemetry import OpenTelemetryConfig
+from starlite.status_codes import HTTP_200_OK
+
+
+def create_config(**kwargs: Any) -> Tuple[OpenTelemetryConfig, InMemoryMetricReader, InMemorySpanExporter]:
+    """Create OpenTelemetryConfig, an InMemoryMetricReader and InMemorySpanExporter.
+
+    Args:
+        **kwargs: Any config kwargs to pass to the OpenTelemetryConfig constructor.
+
+    Returns:
+        A tuple containing an OpenTelemetryConfig, an InMemoryMetricReader and InMemorySpanExporter.
+    """
+    resource = Resource(attributes={SERVICE_NAME: "starlite-test"})
+    tracer_provider = TracerProvider(resource=resource)
+    exporter = InMemorySpanExporter()  # type: ignore
+    tracer_provider.add_span_processor(BatchSpanProcessor(exporter, schedule_delay_millis=1))
+
+    aggregation_last_value = {Counter: ExplicitBucketHistogramAggregation()}
+    reader = InMemoryMetricReader(preferred_aggregation=aggregation_last_value)  # type: ignore
+    meter_provider = MeterProvider(resource=resource, metric_readers=[reader])
+    set_meter_provider(meter_provider)
+
+    meter = get_meter_provider().get_meter("starlite-test")
```

Comment:
You can do this instead of setting global for this test.
```suggestion

    meter = meter_provider.get_meter("starlite-test")
```